controllers/hyperconverged/hyperconverged_controller.go

```diff
@@ -262,6 +262,27 @@ type ReconcileHyperConverged struct {
 	monitoringReconciler *alerts.MonitoringReconciler
 }
 
+func (r *ReconcileHyperConverged) reconcilePolicyConfigMap(req *common.HcoRequest) error {
+
+	configMap := corev1.ConfigMap{}
+
+	if req.Instance.Spec.TuningPolicy == hcov1beta1.HyperConvergedTuningPolicyConfigMap {
+
+		configMapNameSpace := types.NamespacedName{
+			Namespace: req.Instance.Namespace,
+			Name:      string(hcov1beta1.HyperConvergedTuningPolicyConfigMap),
+		}
+		err := r.client.Get(req.Ctx, configMapNameSpace, &configMap)
+
+		if err != nil {
+			return err
+		}
+		// If the configMap has changed kubevirt object have to be updated, otherwise the cache object is used
+		r.operandHandler.Reset()
+	}
+	return nil
+}
+
```

Comment:
please consider moving this logic to the `operands` package. maybe in the kubevirt.go file